packages/synapse-client/package.json

```diff
@@ -37,8 +37,7 @@
     "get-spec:staging": "npx tsx src/codegen/downloadSpec.ts --stack staging && pnpm format-spec",
     "get-spec:production": "npx tsx src/codegen/downloadSpec.ts --stack production && pnpm format-spec",
     "get-spec": "pnpm get-spec:production",
-    "configure": "openapi-generator-cli version-manager set 7.8.0",
```

Comment:
note the current versions is saved in openapitools.json, so this step is unnecessary.